controller/filehandlingcontroller.php

```diff
@@ -143,7 +146,15 @@ public function save($path, $filecontents, $mtime) {
 				// File same as when opened, save file
 				if($this->view->isUpdatable($path)) {
 					$filecontents = iconv(mb_detect_encoding($filecontents), "UTF-8", $filecontents);
-					$this->view->file_put_contents($path, $filecontents);
+					try {
+						$this->view->file_put_contents($path, $filecontents);
+					} catch (LockedException $e) {
+						return new DataResponse(['message' => $e->getMessage()], Http::STATUS_BAD_REQUEST);
```

Comment:
Oh well, you intentionally kill all info. For locking this is quite bad. It failed to save for me just to make me realise, that I had a stray lock on the file....

well we should at least catch the exception and return an error string. But yeah maybe  kill the original one and use a custom text...

but e.g. for the firewall errors (`ForbiddenException`) it would be useful to display the error, so people know saving failed because of that.